packages/web-components/src/components/image/image.ts

```diff
@@ -62,15 +62,25 @@ class DDSImage extends StableSelectorMixin(LitElement) {
   @property({ attribute: 'default-src' })
   defaultSrc = '';
 
+  /**
+   * Whether or not to apply a border around the image.
+   */
+  @property({ type: Boolean, reflect: true })
+  border = false;
+
   render() {
     const { alt, defaultSrc, _images: images, _handleSlotChange: handleSlotChange } = this;
+    const imgClasses = [`${prefix}--image__img`, this.hasAttribute('border') ? `${prefix}--image__img--border` : null]
```

Comment:
Thank you, @jkaeser!
Would you please change this set of classes to use the `classMap` utility?
<img width="769" alt="Screen Shot 2021-09-01 at 12 14 40" src="https://user-images.githubusercontent.com/42848561/131697404-de84d465-b592-49d0-b423-bad51a874ac9.png">